packages/sitecore-jss-react-native/package.json

```diff
@@ -64,8 +64,13 @@
     "ts-jest": "~26.0.0",
     "typescript": "~4.3.5"
   },
-  "resolutions": {
-    "babel-core": "6.26.3"
```

Comment:
babel-core is not needed anymore? was it here because of security vulnerability?

Reply:
no, babel-core was here before that. I removed it because 'resolutions' work only in root package.json, see https://yarnpkg.com/configuration/manifest : "The resolutions field can only be set at the root of the project, and will generate a warning if used in any other workspace."